scripts/rclone_update_nfo_titles.py

```
@@ -84,6 +92,53 @@ def normalize_classification(classification: str) -> str:
     return "-".join(part for part in (mosaic_out, subtitle_out) if part)
 
 
+def parse_year_dir(name: str) -> Optional[int]:
+    """返回目录名表示的年份；非纯数字年份目录返回 ``None``。"""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `；` (FULLWIDTH SEMICOLON). Did you mean `;` (SEMICOLON)?

(RUF002)